sklearn/preprocessing/_encoders.py

```diff
@@ -699,6 +699,26 @@ class OrdinalEncoder(_BaseEncoder):
     dtype : number type, default np.float64
         Desired dtype of output.
 
+    missing_value : object type or a list of object types, default "nan"
+        Which value(s) passed to ``transform()`` to be considered missing
+
+    handle_missing : string, default 'ignore'
+        How to handle missing values passed to ``transform()``
+
+        - 'ignore' : Allow missing values to pass through ``transform()`` and
+           and be returned as np.nan
+        - 'largest' : Encode missing values as largest ordinal category
+        - 'smallest' : Encode missing values as 0
+        - 'separate' : Encode missing values as -1
```

Comment:
Maybe just allow the user to give an integer. I think handle_missing=-1 is clearer... Or maybe not?

Reply:
Definitely. I could see that being useful.